examples/wasm/wasm_sample/Makefile

```diff
@@ -0,0 +1,101 @@
+LLVM_VERSION = 11
```

Comment:
Debian and Ubuntu provide multiple LLVM_VERSIONs in parallel and one can install them in parallel (sometime it is required because some version of llvm is linked into another library (e.g.: graphics))